### DISCUSSION TITLE
Where is FFMPEG file in Heroku

### DISCUSSION THREAD

**Discussion by arisprung18** [jonathanong/heroku-buildpack-ffmpeg-latest#44] [Q&A]
I am directing the FFMPEG to my local file when running on my local computer.
 
```java
FFmpeg ffmpeg = new FFmpeg("/usr/local/Cellar/ffmpeg/4.2.1_2/bin/ffmpeg");
```

However when I deploy to Heroku where should I direct the FFMPEG initialization?

**Comment by spookyuser** [comment]
AFAIK, with this buildpack, `ffmpeg` becomes available as an environmental varible, meaning you should be able to use it anywhere without specifiying the path and just writing `ffmpeg`.

So the path would just be `ffmpeg` and your code would become: 
```java
FFmpeg ffmpeg = new FFmpeg("ffmpeg");
```

But I could be wrong 😥

**Comment by jonathanong** [comment]
on heroku, it's in your `$PATH`, so you just have to call `ffmpeg` from the CLI.

**Comment by jonathanong** [comment]
which i'm assuming is what @spookyUnknownUser's code does
